src/domain/Chain.js

```diff
@@ -0,0 +1,40 @@
+class Chain {
+	static DEFAULT_CHAIN_NAME = Object.freeze('polygon')
+	static Ethereum = Object.freeze(new Chain("ethereum"))
+	static Polygon = Object.freeze(new Chain("polygon"))
```

Comment:
It would be good if we didn't need to update this class when there are new chains. The logic should be: if there's an entry in the config package for a chain, then it's a 'known' chain.

If that's not easy to accomplish, then let's at least add support for `gnosis` here from the start.